package.json

```diff
@@ -8,16 +8,15 @@
     "autogen": "nodemon ./example/autogenerated/autogen.js --exec babel-node -e js",
     "start-middleware": "nodemon ./example/apollo-server/movies-middleware.js --exec babel-node -e js",
     "start-typedefs": "nodemon ./example/apollo-server/movies-typedefs.js --exec babel-node -e js",
-    "build": "babel src --presets babel-preset-env --out-dir dist",
-    "build-with-sourcemaps": "babel src --presets babel-preset-env --out-dir dist --source-maps",
+    "build": "babel src --presets @babel/preset-env --out-dir dist",
+    "build-with-sourcemaps": "babel src --presets @babel/preset-env --out-dir dist --source-maps",
     "precommit": "lint-staged",
     "prepublish": "npm run build",
-    "pretest": "npm run build",
-    "test": "nyc --reporter=lcov ava test/unit/ --verbose",
+    "test": "nyc --reporter=lcov ava test/unit/**.test.js --verbose",
     "parse-tck": "babel-node test/helpers/tck/parseTck.js",
     "test-tck": "nyc ava --fail-fast test/tck/",
     "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "test-all": "nyc ava test/ --verbose",
+    "test-all": "nyc ava test/**/*.test.js --verbose",
```

Comment:
This change misses the tests in `test/unit/neo4j-schema`. What was the reason for the change?

Reply:
This is a syntaks change inside ava due to new version (updated ava from 0.25 to 2.2)

Reply:
The test/**/*.test.js should include all *.test.js within the directory test (also subdirectories)

Reply:
Looks like I was seeing inconsistencies handling the glob pattern.